contributor-docs/versioning.md

```diff
@@ -0,0 +1,131 @@
+that we ship. From semver.org, this means that:
+
+> Given a version number MAJOR.MINOR.PATCH, increment the:
+>
+> 1. **MAJOR** version when you make incompatible API changes,
+> 2. **MINOR** version when you add functionality in a backwards compatible
+>    manner, and
+> 3. **PATCH** version when you make backwards compatible bug fixes.
+>
+> _Additional labels for pre-release and build metadata are available as
+> extensions to the MAJOR.MINOR.PATCH format._
+
+As a result, whenever you see a `minor` or `patch` update for a package from the
+Primer you should feel confident that you can update without
+anything breaking in your project. For a full list of changes that the team
+commits to being backwards-compatible, view the [changes](#changes) table
+below.
+
+## Changes
+
+| Category   | Type of change                                                | semver bump |
+| :--------- | :------------------------------------------------------------ | :---------- |
+| Component  | A component is added                                          | `minor`     |
+|            | A component is deprecated                                     | `minor`     |
+|            | A component is removed                                        | `major`     |
+| Props      | A prop is added to a component                                | `minor`     |
+|            | The type of a prop is made more general                       | `minor`     |
+|            | The type of a prop is made more specific                      | `major`     |
+|            | A prop is deprecated                                          | `minor`     |
+|            | A prop is removed from a component                            | `major`     |
+|            | The element to which additional props are added to is changed |             |
+| Markup     | The DOM node that an `id` corresponds to is changed           |             |
+|            | The DOM node that an `aria-label` corresponds to is changed   |             |
+|            | The `role` of a component is changed                          |             |
+| Styles     | The `position` of the outermost element is updated            |             |
+|            | The `display` property of the outermost element is updated    |             |
+|            | A flex or grid property is updated                            |             |
+|            | A selector is added                                           |             |
+|            | A selector is removed                                         |             |
+|            | The specificity of a selector is raised                       |             |
+|            | The specificity of a selector is lowered                      |             |
+| Behavior   | Interactions are added to a component                         |             |
+|            | Interactions are removed from a component                     |             |
```

Comment:
Do you think changes related to responsiveness can go under this?

Reply:
Ooo, I like that a ton. Great suggestion 🙌